src/Sulu/Bundle/WebsiteBundle/Sitemap/XmlSitemapRendererInterface.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\WebsiteBundle\Sitemap;
+
+use Sulu\Component\Webspace\Portal;
+
+/**
+ * Interface for sitemap-renderer.
+ */
+interface XmlSitemapRendererInterface
```

Comment:
I think I would pass the `PortalInformation` to the methods here. This class already knows about the Sulu internals, so I think there is no harm in passing the `PortalInformation`. And as a positive side effect the method signatures shrink.

Or are these methods called in a context where no `PortalInformation` is passed?